CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 ## Wazuh v4.2.0 - Kibana 7.10.0 , 7.10.2 - Revision 4201
 
+### Added
+
+- Add logtest PUT example on API Console [#3061](https://github.com/wazuh/wazuh-kibana-app/pull/3061)
+
+### Fixed
 
+- Fix wrong variable declaration for macOS agents [#3066](https://github.com/wazuh/wazuh-kibana-app/pull/3066)
 
 ## Wazuh v4.1.3 - Kibana 7.10.0 , 7.10.2 - Revision 4104
```

Comment:
Why has the `CHANGELOG.md`  file 2 not related changes?

Reply:
Mmmm maybe when I applied rebase... let me check